apps/api-extractor/src/api/ExtractorConfig.ts

```diff
@@ -1032,6 +1058,11 @@ export class ExtractorConfig {
             tokenContext
           );
         }
+
+        tagsToReport = {
+          ...defaultTagsToReport,
+          ...apiReportConfig.tagsToReport
```

Comment:
@Josmithr 

What happens if you DON'T want a certain tag to get reported? If I'm reading this code right, the config file can only add new tags, never remove a default tag.

Reply:
(Never mind; I didn't realize this is a `string` ->` boolean` Map)